server/routes/v1/upload.ts

```diff
@@ -107,7 +107,7 @@ export const postUpload = [
           metadata: metadata,
         })
       } catch (err: any) {
-        if (err.toString().includes('duplicate key error')) {
+        if (err.toString().includes('Duplicate version name for model')) {
```

Comment:
We should be testing the error status code, instead of the text.

Reply:
Resolved